pages/importance.md

```diff
@@ -63,32 +63,31 @@ Thus, the purpose of the guide is to teach how to protect publicly available inf
 
-### Анализируем угрозы
+### Analyzing threats
 
-В соответствии с процессом OPSEC проанализируем, кто может быть заинтересован в подобных действиях и как может использовать информацию о нас. Давайте рассмотрим популярные случаи использования первичных данных на схеме ниже и попробуем сделать выводы. 
+According to the OPSEC process, let's analyse who might be interested in such actions and how information about us might be exploited. In the diagram below, let's look at popular uses of primary data and try to draw conclusions. 
 
-![Что можно сделать с нашими данными?](../img/0869d0a1e60173af48378.png)
+![What can be done with our data?](../img/0869d0a1e60173af48378.png)
 
-Что можно сделать с нашими данными?
-Наверняка вы обратили внимание, что большинство пунктов связано с **получением денег**. Думаю, финансовая мотивация мошенников не требует объяснения. Единственное, что следует отметить — деньги могут быть похищены не только у вас, но и у других лиц. В этом случае критичным для нас становится участие в уголовном процессе, куда могут привлечь как свидетелем, так и обвиняемым. 
+What can be done with our data?
+You have probably noticed that most entries are related to **gaining money**. I think the financial motivation of the scammers is self-explanatory. The only issue to note is that money can be stolen from you and others. В этом случае критичным для нас становится участие в уголовном процессе, куда могут привлечь как свидетелем, так и обвиняемым. 
 
-Выгода от сбора полной информации о вас, включая доступ к перепискам, менее определена. Это может быть как бытовое преследование, так и шпионаж, шантаж и прочее. Ясно только, что в большинстве случаев это **причинение личного вреда и вторжение в частную жизнь**.
+The benefits of harvesting comprehensive information about you, including access to correspondence, are less certain. It can be domestic harassment, espionage, blackmail, and so on. In most cases, it is clearly a **personal harm and invasion of privacy**..
 
-Эти угрозы понятны и объяснимы, а во многих случаях регулируются законом. Статьи, связанные с финансовыми потерями, перечислять нет смысла, настолько их много; а сбор личных данных — это 137 УК РФ. Однако, давайте взглянем правде в лицо: требуется серьёзная причина, чтобы идти и самостоятельно форсировать возбуждение дела, а участие в таких процессах — стресс. Поэтому вернёмся к уже озвученной выше позиции: наша цель в том, чтобы защитить свои данные и избежать угроз.
+These threats are clear and explainable and, in many cases, regulated by law. There is no point in listing the articles related to financial loss. There are so many of them, and the collection of personal data is 137 of the Criminal Code. However, let's face it: it takes a severe reason to go and force the case yourself, and getting involved in such processes is stressful. So back to the position already voiced above: our goal is to keep our data safe and avoid threats.
```

Comment:
Articles should be legal acts. And may be it is needs to be clarified that we speak about local russian laws?

Reply:
yes it's better:
what about "137 of the [Russian Criminal Code](Russian Criminal Code)."?